src/parallel_ops/replicate.cc

```diff
@@ -50,7 +50,7 @@ bool ReplicateParams::is_valid(ParallelTensorShape const &input) const {
 
 ReplicateParams Replicate::get_params() const {
   ReplicateParams params;
-  params.replicate_legion_dim = this->replicate_legion_dim;
```

Comment:
Why this change?

Reply:
The original was an error, actually. In `replicate.h`, it is `replicate_dim` and not `replicate_legion_dim`. The nomenclature is actually a little inconsistent, when `[op]_legion_dim` is used vs `[op]_dim`. I've kept the params consistent with the constructor.